web-admin/src/features/dashboards/query-mappers/mapQueryToDashboard.ts

```diff
@@ -107,11 +103,31 @@ export function mapQueryToDashboard(
       }
 
       if (validSpecResp.error || timeRangeSummary.error) {
-        // error state
         set({
           isFetching: false,
           error:
-            validSpecResp.error?.message ?? timeRangeSummary.error?.message,
+            validSpecResp.error?.response?.data?.message ??
+            timeRangeSummary.error?.response?.data?.message,
+        });
+        return;
+      }
+
+      if (
```

Comment:
There are suppressing underlying error message to the user. Do we have an alternative to get them when the user reports it? Else better to append the actual message at the end.

Reply:
Ah I missed that